qt/scientific_interfaces/CMakeLists.txt

```diff
@@ -64,8 +64,6 @@ set(TEST_FILES
     test/ISISReflectometry/Save/SavePresenterTest.h
     test/ISISReflectometry/Common/ClipboardTest.h
     test/ISISReflectometry/Common/PlotterTestQt4.h
-    test/ISISReflectometry/Common/DecoderTest.h
-    test/ISISReflectometry/Common/EncoderTest.h
```

Comment:
How come these have been removed, is this related to another PR or a mistake?

Reply:
Nope these this is intentional, these tests were still running in qt4 mode, which didn’t support the qt5 backend used for the mantidcanvas class defined here.